app/views/projects/new.html.erb

```diff
@@ -6,6 +6,13 @@
   Before making a new project here, be sure to upload the project to the servers if the files are not already there. <br>
   You can follow the directions on the <a href="https://github.com/OSC/frame-renderer/#upload-your-project">README</a> on how to do so.
   <br><br>
+
+  <% unless ENV["SFTP_HOST"].nil? %>
```

Comment:
this should be added as a method to https://github.com/OSC/frame-renderer/blob/master/config/configuration_singleton.rb, and use `if` instead of `unless nil` so that then you can do something like this:

```
<% if Configuration.sftp_host %>
  ...
<% end %>
```